src/status_im/contexts/wallet/events.cljs

```diff
@@ -345,8 +348,7 @@
                            :Prod
                            data-store/rpc->network)
                      data)}]
-     {:fx [[:dispatch [:wallet.tokens/get-token-list]]]
-      :db (assoc-in db [:wallet :networks] network-data)})))
+     {:db (assoc-in db [:wallet :networks] network-data)})))
```

Comment:
is there any kind of dependency between the `:wallet/get-ethereum-chains` and `:wallet.tokens/get-token-list` events? I would assume that the `:wallet/get-token-list` event was dispatched in `:wallet/get-ethereum-chains-success` due to that.

Reply:
I found out there is a dependency (chain-ids based on testnet mode) on the success of the token-list event. I have reverted this change. It seems that the previous change in this PR worked as well because of the response times of both events -`get-ethereum-chains` (very quick) and `get-token-list` (takes a bit of time). Anyhow, this works as expected.